deepfence_server/model/scans.go

```diff
@@ -557,3 +557,9 @@ func (v CloudCompliance) GetCategory() string {
 func (CloudCompliance) GetJsonCategory() string {
 	return "severity"
 }
+
+type VulnerabilityComparison struct {
+	New     []Vulnerability `json:"new" required:"true"`
+	Updated []Vulnerability `json:"updated" required:"true"`
+	Deleted []Vulnerability `json:"deleted" required:"true"`
```

Comment:
What does Updated & Deleted mean?
It should only be 2 sets, diff & common
A vulnerability cannot change, it can either disappear or stay there.

Reply:
- Updated: Right now not used. Vulnerability can change. (Score, severity, etc can change in the source database)
- Deleted: Mitigated vulnerabilities compared to other scan
- New: New vulnerabilities compared to other scan

From user's point of view, `new` and `deleted` are the important ones.

Reply:
I have kept `updated` for now to use it later in future in case we need to use for cvss score, etc.